universal-login-commons/src/integration/http/CoingeckoApi.ts

```diff
@@ -45,7 +45,9 @@ export class CoingeckoApi {
   };
 
   fetchTokenInfo = async (tokenDetails: TokenDetailsWithCoingeckoId[], currencies: ObservedCurrency[]) => {
-    const tokens = tokenDetails.map(token => token.coingeckoId);
+    const tokens = tokenDetails
+      .filter((token, index, tokens) => tokens.findIndex(t => t.symbol === token.symbol) === index)
```

Comment:
why?

Reply:
We can't have duplicated tokens  when we call api

Reply:
So then could we please extract function `removeDuplications`?

Reply:
We can :))